.github/workflows/test-and-report.yml

```diff
@@ -38,4 +38,4 @@ jobs:
                 file: ./coverage.xml
                 flags: unittests
                 name: codecov-umbrella
-                fail_ci_if_error: true
```

Comment:
For some reason, this action is failing to upload the file to codecov. Looking at the page for the action (https://github.com/marketplace/actions/codecov), they now recommend using @v1.5.0 and switching "file" to "files". Want to give that a shot to see if that fixes this issue?